osbs/api.py

```diff
@@ -413,10 +413,29 @@ def create_prod_build(self, git_uri, git_ref,
         build_request = self.get_build_request()
         labels = utils.Labels(df_parser.labels)
 
+        required_missing = False
         try:
             _, name_value = labels.get_name_and_value(utils.Labels.LABEL_TYPE_NAME)
+        except KeyError:
```

Comment:
I think it is - the methods are fast, and the clarity of the code is more important in this case. 

@twaugh, @lcarva thoughts?